extensions/binding/org.eclipse.smarthome.binding.homematic/README.md

```diff
@@ -223,6 +225,15 @@ The disadvantage is of course, that all events for this channel are delayed.
 ```
 
 The Type is the device type, channel number and lowercase channel name separated with a underscore.
+If you are using Homegear, the ```HG-``` prefix is not needed for the Type.
+
+Example without channel configs
```

Comment:
I think that it is good to have an example without channel configs. Suggestion: Move the new sentence "The channel configs are optional." directly above this example, this might improve reading flow here.